doc/classes/Basis.xml

```diff
@@ -4,9 +4,9 @@
 		A 3×3 matrix for representing 3D rotation and scale.
 	</brief_description>
 	<description>
-		A 3×3 matrix used for representing 3D rotation and scale. Usually used as an orthogonal basis for a [Transform3D].
-		Contains 3 vector fields X, Y and Z as its columns, which are typically interpreted as the local basis vectors of a transformation. For such use, it is composed of a scaling and a rotation matrix, in that order (M = R.S).
-		Basis can also be accessed as an array of 3D vectors. These vectors are usually orthogonal to each other, but are not necessarily normalized (due to scaling).
+		The [Basis] built-in [Variant] type represents a 3x3 [url=https://en.wikipedia.org/wiki/Matrix_(mathematics)]matrix[/url] used for 3D rotation, scale, and shearing. It is frequently used within a [Transform3D].
+		?? A [Basis] is composed by 3 axis vectors: [member x], [member y], and [member z]. The length of each axis ([method Vector3.length]) influences the basis' scale. Usually, these axes are perpendicular to one another. If this is not the case, the basis is sheared and is thus not comformal (see [method is_conformal]). Applying this basis to a 3D model will result in the model appearing distorted.
```

Comment:
> ##### This PR may contain multiple versions of the same line I was not sure about, denoted with "??". Reviewing of these lines is heavily appreciated.

I'll bring it right on top of the PR's description next time.
Which ones do you prefer or do you have more suggestions?

Reply:
When noting the length of the vector of basis matrices, it is necessary to specify that the right-hand orthogonal system is positive. This is because focusing only on length can lead to negative scales.

Reply:
@TokageItLab If you have more insight like this on where it is absolutely necessary to mention these terms please do highlight them to make this PR as right as it can be